1-exercises/A-accessing-values/exercise3.js

```diff
@@ -20,8 +20,10 @@ let basketballTeam = {
     - console.logs the name of each player on a new line
 */
 
-// write code here
 
+let topPlayers = ["Michael Jordan", "Scottie Pippen", "Dennis Rodman"];
```

Comment:
You should not copy the code but use the dot notation or bracket notation to retrieve values from the object. 
Please check this:
https://codeburst.io/javascript-quickie-dot-notation-vs-bracket-notation-333641c0f781
https://syllabus.codeyourfuture.io/js-core-2/week-1/lesson#objects